package.json

```diff
@@ -2,22 +2,39 @@
   "name": "@mapbox/fontmachine",
   "version": "0.2.2",
   "description": "Make GL-ready pbfs and metadata for usage in fontstack API.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mapbox/fontmachine.git"
+  },
   "main": "index.js",
   "scripts": {
-    "test": "jshint *.js & tap --coverage test/test.js",
-    "docs": "dox -r < index.js | doxme --readme > README.md"
```

Comment:
I got rid of the `docs` script because unfortunately `dox` doesn't really work that well and its output requires some hand-massaging. I figured we should just stick to manually maintaining the documentation for this small project.